kolibri/content/api.py

```diff
@@ -0,0 +1,222 @@
+"""
+This module acts as the only interface point between other apps and the database backend for the content.
+It exposes several convenience functions for accessing content
+"""
+from kolibri.content import models as KolibriContent
+from uuid import UUID
+import os
+from bulk_update.helper import bulk_update
+from django.core.files import File as DjFile
+
+"""helper funcitons"""
```

Comment:
Can you elaborate on this a bit? 
the helper function is_valid_uuid use UUID which is imported from uuid. How can I avoid to import UUID if I move is_valid_uuid to utils?